collections/_article/dev-snapshot-godot-4-2-dev-1.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "Dev snapshot: Godot 4.2 dev 1"
+excerpt: "We are ready to share the first batch of improvements coming to Godot 4.2 later this year with the release of the first dev snapshot!"
+categories: ["pre-release"]
+author: Yuri Sizov
+image: /storage/blog/covers/dev-snapshot-godot-4-2-dev-1.webp
+image_caption_title: "Terrain3D"
+image_caption_description: "A tool for 3D terrain creation by Outobugi Games and Tokisan Games"
+date: 2023-07-19 15:00:00
+---
+
+Two weeks ago we wrapped up the development of [Godot 4.1](/article/godot-4-1-is-here), bringing more polish and stability to the Godot 4 series, and just two days ago we shipped its [first maintenance release](/article/maintenance-release-godot-4-1-1), addressing some of the more urgent issues that slipped in. In preparation to both of these releases many improvements were temporarily put aside to ensure stability and lack of last minute regressions.
```

Comment:
```suggestion
Two weeks ago we wrapped up the development of [Godot 4.1](/article/godot-4-1-is-here), bringing more polish and stability to the Godot 4 series, and just two days ago we shipped its [first maintenance release](/article/maintenance-release-godot-4-1-1), addressing some of the more urgent issues that slipped in. In preparation for both of these releases we temporarily put aside many improvements to ensure stability and to avoid any last minute regressions.
```